---
title: ""
bibliography: 
  - references.bib
  - references_other.yaml
mainfont: Roboto-Regular.ttf
fontsize: 12pt
mainfontoptions: 
  - BoldFont=Roboto-Bold.ttf
  - ItalicFont=Roboto-Italic.ttf
  - BoldItalicFont=Roboto-BoldItalic.ttf
output: 
  bookdown::pdf_document2:
    citation_package: default # Use pandoc default, otherwise csl can't be used for reference formatting
    toc: no
    number_sections: no
    fig_caption: yes
    keep_tex: yes # for converting tex to doc
    latex_engine: xelatex
    pandoc_args: [ "--csl", "journal-of-plant-research.csl"]
    includes:
      in_header: journal-of-plant-research.sty
# Prevent figures from moving around, suppress automatic "Figure #:" in captions
header-includes: 
  \usepackage{float}
  \makeatletter\renewcommand*{\fps@figure}{H}\makeatother
  \renewcommand{\caption}{}
params:
  output_type: "ms"
editor_options: 
  markdown: 
    wrap: sentence
  chunk_output_type: console
---

```{r ms-setup, include = FALSE, cache = FALSE}
knitr::opts_chunk$set(
  echo = FALSE,     # Don't display the source code
  message = FALSE,  # Don't preserve messages
  warning = FALSE,  # Don't preserve warnings 
  results = "hide", # Hide text output
  cache = FALSE    # Don't use the cache
  )
```

```{r load-data}
tar_load(refs)
tar_load(refs_other)
tar_load(climate)
tar_load(filmy_species)
tar_load(moorea_sites)
tar_load(phylosig)
tar_load(range_comparison)
tar_load(combined_species_means)
tar_load(t_test_results)
tar_load(env_range_dt_model_summary)
tar_load(recovery_species_means)
tar_load(filmy_habit)
tar_load(filmy_phy)
tar_load(mean_vpd_sporo)
tar_load(mean_vpd_gameto)
tar_load(glmm_summary)
tar_load(glmm_params)
tar_load(env_range_dt_model)
tar_load(env_range_dt_model_summary)
tar_load(filmy_dt_chamber)
tar_load(rel_water_species_means)
```

<!-- to move to the next line without starting a new paragraph, 
add two spaces after last word, e.g., Joel Nita__ -->

```{r first-page, results = "asis", eval = params$output_type == "ms", purl = FALSE}
cat("
**Corresponding author**  
Joel H. Nitta  
Department of Integrated Biosciences, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan  
Phone: +81-4-7136-3681  
Fax: +81-4-7136-3680  
email: joelnitta\\@gmail.com

**Membership of the Botanical Society of Japan**  
YES (Joel H. Nitta)

**Subject area**  
(1) Ecology/Ecophysiology/Environmental Biology

**Number of tables, black-and-white figures, and color figures**  
4 tables / 0 black-and-white figures / 4 color figures

**Author contributions**  
Joel H. Nitta conceived of and designed the study with input from James E. Watkins, N. Michele Holbrook, and Charles C. Davis. Data collection was performed by Joel H. Nitta and Tristan W. Wang. Data analysis was performed by Joel H. Nitta. The first draft of the manuscript was written by Joel H. Nitta, with subsequent comments and edits from the other authors. All authors read and approved the final manuscript.

\\newpage
")
```

# Intergenerational niche differentiation in filmy ferns (Hymenophyllaceae)

Joel H. Nitta^1,4^, James E. Watkins, Jr.^2^, N. Michele Holbrook^1^, Tristan W. Wang^1,3^, and Charles C. Davis^1^

^1^Department of Organismic and Evolutionary Biology and Harvard University Herbaria, Harvard University, Cambridge, Massachusetts 02138 USA

^2^Department of Biology, Colgate University, 13 Oak Drive, Hamilton, New York 13346 USA

^3^Current affiliation: Section of Plant Pathology and Plant-Microbe Biology, Cornell University, Ithaca, New York 14853 USA

^4^Current affiliation: Department of Integrated Biosciences, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan  

Author for correspondence: Joel Nitta (joelnitta\@gmail.com)

ORCID ID numbers  
Joel H. Nitta: \textnormal{https://orcid.org/0000-0003-4719-7472}  
N. Michele Holbrook: \textnormal{https://orcid.org/0000-0003-3325-5395}  
Tristan W. Wang: \textnormal{https://orcid.org/0000-0001-6889-2453}  
Charles C. Davis: \textnormal{https://orcid.org/0000-0001-8747-1101}  

### Abstract

```{r study-system-stats}
# Calculate number of all filmy fern species, trichomanoids, and hymenophylloids
# (not considering numbered taxa, ie, varieties)
n_filmy_sp <- str_remove_all(filmy_species, "[0-9]") %>% 
  n_distinct()

n_trich_sp <- str_remove_all(filmy_species, "[0-9]") %>% 
  magrittr::extract(., str_detect(., "^Hymeno", negate = TRUE)) %>%
  n_distinct()

n_hymeno_sp <- str_remove_all(filmy_species, "[0-9]") %>% 
  magrittr::extract(., str_detect(., "^Hymeno", negate = FALSE)) %>%
  n_distinct()
```

Desiccation tolerance was a key trait that allowed plants to colonize land.
However, little is known about the transition from desiccation tolerant non-vascular plants to desiccation sensitive vascular ones.
Filmy ferns (Hymenophyllaceae) represent a useful system to investigate how water-stress strategies differ between non-vascular and vascular stages within a single organism because they have vascularized sporophytes and nonvascular gametophytes that are each capable of varying degrees of desiccation tolerance.
To explore this, we surveyed sporophytes and gametophytes of `r n_filmy_sp` species (`r n_distinct(filmy_species)` taxa including varieties) of filmy ferns on Moorea (French Polynesia) and used chlorophyll fluorescence to measure desiccation tolerance and light responses.
We conducted phylogenetically informed analyses to identify differences in physiology between life stages and growth habits.
Gametophytes had similar or less desiccation tolerance (ability to recover from 2 d desiccation at -86 MPa) and lower photosynthetic optima (maximum electron transport rate of photosystem II and light level at 95% of that rate) than sporophytes.
Epiphytes were more tolerant of desiccation than terrestrial species in both generations.
Despite their lack of greater physiological tolerances, gametophytes of several species occurred over a wider elevational range than conspecific sporophytes.
Our results demonstrate that filmy fern gametophytes and sporophytes differ in their physiology and niche requirements, and point to the importance of microhabitat in shaping the evolution of water-use strategies in vascular plants.

**Keywords**: chlorophyll fluorescence; desiccation tolerance; DNA barcoding; French Polynesia; gametophyte; Hymenophyllaceae

\newpage

## Introduction

The movement of plants from water to land was one of the most important evolutionary events in the history of the earth, fundamentally altering the global carbon and water cycle and setting the stage for modern terrestrial ecosystems [@Graham1993; @Kenrick1997a].
The transition from aquatic to terrestrial growth was complex, requiring a suite of evolutionary adaptations to survive in deadly dry habitats bombarded with radiation [@Waters2003].
Lacking mechanisms to regulate water loss, the earliest land plants were likely poikilohydric, \ie, they existed in a state of equilibrium with the humidity of the external environment [@Raven1999].
Therefore, one particularly important early adaptation that enabled plants to survive on land was desiccation tolerance (DT), the ability to lose all metabolically active water, reach equilibrium with atmospheric humidity, and recover upon rewetting [@Mishler1985].

Compared with the ubiquitous DT among the earliest land plants, the occurrence of DT varies across extant land plant lineages and life stages.
Many bryophytes have retained vegetative DT in the gametophyte stage [@Oliver2005, and references therein].
In extant vascular plants (tracheophytes), DT occurs in certain non-vegetative stages (\eg, seeds and pollen), but has largely been lost in vegetative tissues [with some notable exceptions, the so-called 'resurrection plants'; @Hartung1998; @Kappen2007; @Scott2000].
Instead, the majority of tracheophytes solely rely on avoiding desiccation with adaptations including a waxy cuticle, stomata, roots, and an extensive vascular system.
Correlated with these differences in functional strategies to cope with water stress is a major difference in life cycle.
Over time, plants shifted away from wholly gametophyte-dominated life cycle (as in extant bryophytes) to co-independence of the gametophyte and sporophyte (\ie, ferns and lycophytes) to complete loss of the independent gametophyte (\ie, spermatophytes).
Little is known about the details of the transition from desiccation tolerance to avoidance, nor how this took place in conjunction with the switch from a gametophyte to a sporophyte-dominated life cycle; yet, these two steps underpin the radiation of vascular plants onto dry land [@Puttick2018; @Qiu2006; @Wickett2014].

Ferns represent an important, understudied group of tracheophytes for investigating both the ecology and evolution of desiccation tolerance.
Ferns are the only major lineage among land plants with sporophytes and gametophytes that are capable of growing independently from each other.
These two life stages differ drastically in their morphology and physiology---fern sporophytes have true vascular tissue and are homoiohydric (\ie, they regulate their internal water content), whereas fern gametophytes are much smaller (often < 1 cm), lack vascular tissue, and are poikilohydric.
Recent studies indicate that the gametophytes of many fern species are desiccation tolerant, while sporophytes of the same species are thought to lack DT [@Pittermann2013; @WatkinsJr.2007c].
Thus, ferns present a unique opportunity to observe the transition from desiccation tolerance to desiccation avoidance across independently growing life phases within a single organism.

Filmy ferns (Hymenophyllaceae) represent a particularly interesting case for studying the ecological and evolutionary significance of vegetative DT across life stages in ferns.
Filmy ferns are a large family [\circa{ }430 spp.; @PteridophytePhylogenyGroupI2016] of primarily tropical, leptosporangiate ferns named for their remarkably thin leaf laminae, which are usually only a single cell layer in thickness and lack cuticle or stomata (`r figure("habit")`a--d).
Filmy fern sporophytes are a remarkable exception to the 'typical' homoiohydric fern sporophyte described above because they have reverted to poikilohydry: their thin laminae allow for passive, rapid uptake and loss of moisture, thereby maintaining nearly constant equilibrium with atmospheric humidity [@Raven1999].
Many species of epiphytic filmy ferns have extremely reduced root systems [@Schneider2000], relying completely upon absorption through leaf laminae for water [@Shreve1911], although terrestrial species possess true roots and use ground water [@Dubuisson2011].
Thus, the morphological and physiological gap between sporophytes and gametophytes is arguably smaller in filmy ferns compared to any other fern clade.
This makes filmy ferns a potentially useful group for understanding the transition from a desiccation tolerant, gametophyte-dominated strategy to a desiccation avoiding, sporophyte-dominated one, since the sporophytes and gametophytes of ancestral tracheophytes were also likely initially similar, then gradually diverged by a series of step-wise adaptations [@Bateman1998; @Ligrone2012a].

Filmy fern sporophytes encompass a broad diversity of growth forms (terrestrial, saxicolous, hemi-epiphytic, and holoepiphytic) and morphologies, with rhizome and leaf sizes varying over an order of magnitude [@Dubuisson2013; @Hennequin2008; @Nitta2009].
As would be expected from their poikilohydric nature, DT of varying degrees has been reported in sporophytes of many filmy fern species (see references below).
Filmy fern gametophytes are similarly diverse, with morphotypes including filamentous, ribbon, and a combination of these two types (`r figure("habit")`e--h).
While there are no quantitative data about DT in filmy fern gametophytes available, DT has been reported in gametophytes from several other fern lineages [@Pittermann2013; @WatkinsJr.2007a; @WatkinsJr.2012], and may be expected to occur in filmy ferns as well.
The possibility for varying degrees of vegetative DT in both stages of the life cycle of filmy ferns makes them a potentially useful system to compare how DT is utilized between haploid, non-vascular and diploid, vascular life stages.
Furthermore, the diversity of growth forms and habitats in filmy ferns makes them an ideal group for a comparative study of water-stress strategies (\ie, desiccation tolerance vs. desiccation avoidance) and the correlation of these strategies with ecological niche.

The degree of DT in filmy fern sporophytes occupying different niches has been relatively well studied.
In a series of elegant experiments on Jamaican filmy ferns, @Shreve1911 demonstrated that water-use strategies differ between species occupying contrasting habitats (\ie, terrestrial, low elevation epiphyte, and high elevation epiphyte).
Proctor [-@Proctor2003; -@Proctor2012] investigated water relations and the ability to recover from desiccation in sporophytes of several filmy ferns, and found a general correlation between high DT and adaptation for high light levels in widespread species vs. low DT and adaptation to low light levels in species occupying more sheltered habitats.
In the only study to our knowledge including in situ physiological observations of hymenophyllaceous gametophytes, @Johnson2000 found that gametophytes of *Vandenboschia speciosa* (Willd.) G. Kunkel (= *Trichomanes speciosum* Willd.) are adapted for extremely low light environments, and operate at lower light levels than conspecific sporophytes.
@Parra2009 and @Saldana2013 investigated the relationship between physiology (photosynthetic rates and water relations) and vertical stratification in several epiphytic filmy fern species (primarily genus *Hymenophyllum*) along tree trunks in Chile.
They found that species occurring at greater heights in the trees tended to have greater DT, higher photosynthetic capacity (A~max~), and lower rates of evapotranspiration.
A study using two species of *Hymenophyllum* demonstrated that DT in filmy ferns likely operates via a homoiochlorophyllous, constitutive mechanism, which is more similar to that of bryophytes than other drought-tolerant vascular plants, thus allowing filmy ferns to adjust to dry conditions extremely rapidly [@GarcesCea2014].
This was further supported by evidence that induced biomechanical mechanisms likely do not play a role in DT of filmy ferns [@Fallard2018; but see @Mkhize2020].
Recently, transcriptomics has been applied to elucidate the genes involved at each step of the desiccation-recovery cycle of filmy ferns [@Ostria-Gallardo2020; @Ostria-Gallardo2020a]. 
In epiphytic *Hymenoglossum cruentum* Cav., key genes and gene networks were identified related to intracellular mobility and photosynthetic metabolism in the desiccated state, and those related to detoxifying pathways and stabilization of photosystems during rehydration [@Ostria-Gallardo2020a].
In a comparison of low-trunk and canopy epiphytes, @Ostria-Gallardo2020 identified multiple differentially expressed genes that may be specific to tolerating desiccation in these microhabitats.

Despite this considerable interest in filmy fern ecophysiology, with the sole exception of @Johnson2000, no studies have compared physiology between gametophyte and sporophyte phases, and we are aware of none that have done so across multiple species.
Furthermore, to our knowledge, no physiological studies of filmy ferns have accounted for their phylogenetic relationships, and we lack an understanding of how physiological strategies have evolved in this clade.
In addition to DT, photosynthetic optima (adaptation to different light levels) are likely to be important in determining the niches of life stages and species of filmy ferns [@Flores-Bavestrello2016; @Parra2015; @Proctor2003].
Terrestrial and epiphytic species experience different levels of photosynthetic radiation and desiccation, and light intensity varies with height along the host plant in epiphytes [@Benzing1990; @WatkinsJr.2009].
The combined effect of desiccation and light stress may induce greater oxidative damage in filmy ferns than either of these stressors alone [@Niinemets2018], suggesting the need to integrate studies of DT with photosynthesis to gain a full picture of physiological adaptation in filmy ferns.
Here, we leverage a recently developed DNA barcoding system [@Nitta2017] to test several hypotheses related to DT and photosynthesis in a comparative phylogenetic framework including both filmy fern sporophytes and gametophytes: 
1.) Light responses and DT differ between species with different growth habits.
2.) Light responses and DT differ between sporophytes and gametophytes.
3.) DT increases with decreasing environmental water availability.
4.) Gametophytes with broader elevational ranges have higher DT than those with with narrower ranges.

## Materials and methods

### Study system

We selected the island of Moorea, French Polynesia (17°32' S, 149°50' W) for our survey of filmy fern ecophysiology.
Moorea is a small (135 km^2^; maximum elevation 1206 m) tropical oceanic island, located \circa{ }15 km NW of its larger sister island, Tahiti.
Despite its small size, it hosts a range of habitats including coastal strand, low elevation rainforest, and high elevation cloud forest.
A total of `r n_filmy_sp` filmy fern species are known from Moorea, including multiple terrestrial, saxicolous, and epiphytic species [@Florence in press; @Murdock2003; @Nitta2011b; @Nitta2017; @Ranker2005] (\figs{ }`r figure_num("habit")`, `r s_figure_num("tree")`; all supplemental figures available in Online Resource 1).
The two major clades of filmy ferns, the hymenophylloid and trichomanoid lineages [@Pryer2001a], are represented on Moorea by `r n_hymeno_sp` and `r n_trich_sp` species, respectively.
We follow the taxonomic system of @Ebihara2006 for Hymenophyllaceae.

### Field survey and DNA barcoding

We surveyed filmy ferns on Moorea as part of a larger study including all ferns of Moorea and Tahiti [@Nitta2017].
During this study, `r n_distinct(moorea_sites$site)` sampling sites were established from \circa{ }200 m to 1200 m primarily along trails to the summits of three large peaks on Moorea: Mt. Rotui (899 m), Mt. Mouaputa (880 m), and Mt. Tohiea (1206 m).
At each site, sporophytes were sampled in a single 10 m × 10 m plot, and gametophytes sampled in two to three 50 cm × 50 cm plots located within the sporophyte plot.
To estimate abundance of sporophytes, the 10 m × 10 m plot was subdivided into 25 2 m × 2 m subplots, and the presence of species in the sporophyte phase in each subplot was summed such that each species was assigned an abundance rank from 1 to 25.
For the gametophyte plots, each 50 cm x 50 cm plot was subdivided into 25 5 x 5 cm squares, and a single gametophyte selected closest to the center of each square, to later be identified to species using DNA barcoding (see below).
Epiphytes were sampled to a maximum height of \circa{ }2 m on their host plants.
For additional details on sampling procedure, see @Nitta2017.
The growth habit of each species was scored based on field observations as terrestrial, saxicolous, low elevation epiphyte (mostly occurring outside of cloud forest, \ie, below \circa{ }500 m), or high elevation epiphyte (mostly occurring in cloud forest, \ie, above \circa{ }500 m).
*Crepidomanes bipunctatum* (Poir.) Copel. and *C. humile* (G. Forst.) Bosch were observed growing as both epiphytes and saxicoles, but were treated as epiphytes to distinguish them from exclusively saxicolous species.

Unlike sporophytes, fern gametophytes are cryptic and cannot be identified to species based on morphology alone. 
Therefore, we used DNA barcoding to identify filmy fern gametophytes to species following @Nitta2017.
Briefly, DNA extraction was performed using a modified CTAB protocol [@Doyle1987] or the Plant Mini DNEasy kit (Qiagen, Valencia, California, USA) following the manufacturer's protocol.
Chloroplast *rbcL* and *trnH--psbA* were amplified and sequenced using primers and protocols following @Nitta2017.
For each marker, a reference library was constructed from sporophyte sequences using the \function{makeblastDB} command in BLAST [@Altschul1997].
Field-collected gametophytes were identified by a local BLAST query against the reference library.
Gametophytes matching > 99.5 % with a single reference sequence and no others were identified as that species; those that did not match any reference sequence or matched > 99.5 % with multiple reference sequences were not identified and excluded from further analysis.
For this study, only gametophytes that could be identified as Hymenophyllaceae were used; others were excluded from analysis.

Fieldwork was done under permits issued by the French Polynesian Government (Délégation à la Recherche) and the Haut-commissariat de la République en Polynésie Francaise (Protocole d’Accueil 2012–2014). 
Voucher specimens were deposited at UC, with duplicates at GH and PAP (abbreviations follow @Thiers2021).

### Microclimate

```{r microclimate-stats}
climate_start <- climate %>% pull(date) %>% min()
climate_end <- climate %>% pull(date) %>% max()
climate_n_sites <- climate %>% pull(site) %>% n_distinct()
climate_n_days <- climate %>% pull(date) %>% n_distinct()
climate_n_loggers <- climate %>% count(site, habit) %>% nrow()
```

We used microclimate data collected by @Nitta2020.
This dataset includes relative humidity (RH) and temperature measured every 15 min from `r climate_start` to `r climate_end` at the fern survey sites with Hobo ProV2 dataloggers (Onset Computer Corporation, Bourne, Massachusetts, USA), and vapor pressure deficit (VPD) calculated from these two values.
The dataloggers were outfitted with radiation shields (RS1 or RS3, Onset Computer Corporation, Bourne, Massachusetts, USA) to prevent direct contact with precipitation and solar radiation, and placed in pairs at each site (one 'epiphytic' datalogger at \circa{ }1.5 m on a tree, one 'terrestrial' datalogger at \circa{ }10 cm next to the tree).
During the study period, some dataloggers malfunctioned, presumably due to prolonged exposure to high humidity.
Any days during which one or more dataloggers failed are excluded from the dataset.
One terrestrial datalogger (Mt. Tohiea 393 m site) was lacking data for more than half of the survey period, but had data available for the same period from a preliminary survey conducted during the previous year (2012--2013), so the previous year's data for this site was used instead.
The dataset includes `r climate_n_days` days of data from `r climate_n_sites` sites (`r climate_n_loggers` dataloggers).
Mean daily temperature on Moorea is strongly correlated with elevation, dropping \circa{ }6 °C from the lowest surveyed site (201 m) to the highest (1206 m); temperature does not differ significantly between epiphytic and terrestrial habitats [@Nitta2020].
Epiphytic habitats tend to have lower daily mean RH, and greater daily variation in RH, than terrestrial ones.
This difference is greatest at low elevations, and smallest at high elevations [@Nitta2020].

### Desiccation tolerance

Samples were collected in the field and stored in plastic bags with a small amount of water to keep them fresh during transport to the lab.
For sporophytes, eight individuals (one individual = single whole frond including a \circa{ }2 cm section of rhizome) were used per treatment, and all individuals for each species came from a single population.
We did not attempt to differentiate genotypes between fronds in species with long-creeping rhizomes, which often form clonal mats; hence, in some cases, individuals may be genetically identical ramets.
All experiments were initiated within 48 h of collection.
Pre-treatment maximum photochemical yield of photosystem II (F~v~/F~m~) [@Kitajima1975] was measured in fresh plants after a 10 min period of dark-adaptation using a portable mini-PAM chlorophyll fluorometer (Walz Gmbh, Effeltrich, Germany).
Any samples with pre-treatment F~v~/F~m~ < 0.4 were excluded from analysis, as typical values of F~v~/F~m~ in healthy plants tend to be \circa{ }0.8 [@Bjorkman1987].
Samples were then transferred to desiccation chambers containing saturated salts at three different desiccation intensities or a control treatment with moist tissues (100 % RH), and water withheld for either a short (2 d) or long (15 d) interval [@Testo2013].
Conditions inside the desiccation chambers were monitored during the experiment using Track-It RH/Temp dataloggers (Monarch Instrument, Amherst, NH) logging every 10 min (2012 field season for species other than *Callistopteris apiifolia* (Presl) Copel.) or Hobo ProV2 dataloggers logging every 5 min (2012 *C. apiifolia*, 2013, 2014 field seasons).
The desiccation chambers were kept in an air-conditioned room. 
Temperature inside the chambers was mean `r mean(filmy_dt_chamber$temp) %>% round(1)` ± SD `r sd(filmy_dt_chamber$temp) %>% round(2)` °C (`r s_figure("chamber")`) (hereafter, the value following '±' is standard deviation unless otherwise mentioned).
Salts used for desiccation and their corresponding mean water potentials and approximate relative humidity and VPD are as follows: LiCl (-282 MPa, 18 % RH, 2.45 kPa), Mg(NO~3~)~2~ (-86 MPa, 58 % RH, 1.25 kPa), and NaCl (-38 MPa, 80 % RH, 0.60 kPa).
Following the desiccation treatment, plants were rewetted by wrapping them in moist tissues using water from a local stream.
Plants were allowed to rehydrate, and F~v~/F~m~ was again measured at 0.5 h, 24 h, and 48 h following rewetting.
Ability to recover from desiccation was quantified by comparing pre- and post-treatment F~v~/F~m~ [@WatkinsJr.2007c].

We measured relative water content (RWC) in a subset of samples by recording the mass of samples prior to the DT test (fresh mass), at each step of the DT test (turgid mass), then following the DT test after drying them overnight in an oven at 65 °C (dry mass). 
Relative water content was calculated as \(RWC = \frac{TM - DM}{FM - DM} \times 100\), where *TM* is turgid mass, *DM* is dry mass, and *FM* is fresh mass.

Because field-collected gametophytes could not be identified to species prior to DNA extraction, no planned replication by species was possible.
We therefore applied the same treatment for all gametophytes (2 d at -86 MPa) in batches of individuals (`r s_table("gameto-batches")`), and verified the species of each individual after the experiment using the DNA barcode approach described above.
Thus, gametophytes of some species were sampled many times whereas those of other species were not.
Relative water content was not calculated for gametophytes, as these were too small to measure accurately with the balances available.

### Light responses

Rapid light response curves were constructed for each species by measuring photosynthetic yield at gradually increasing levels of photosynthetically active actinic light (400 nm to 700 nm) with the Light Curve function of the mini-PAM portable chlorophyll fluorometer as described by the manufacturer.
Briefly, the maximum yield (F~m~) was first measured in the absence of actinic light.
Next, plants were equilibrated to the actinic light for 30 s, and photosynthetic yield (∆F/F~m~) measured with a 0.8 s saturating pulse.
This was repeated for each photosynthetic photon flux density (PPFD) level, up to ca.
500 µmol·m^-2^·s^-1^.
Response curves were fitted using the equation $y = A(1 - e^{-kx})$, where *y* is relative electron transport rate of photosystem II (RETR), *x* is photosynthetic photon flux density (PPFD), *A* is the asymptote of the curve, and *k* is a slope parameter; outliers at high PPFD likely to distort the curve were discarded [@Proctor2012].
The PPFD at 95 % saturation of RETR (PPFD~95%~) and maximum RETR (ETR~max~) were then calculated from the curve.
Light responses were measured in eight individuals per species in the lab for sporophytes, and on single individuals in the field for gametophytes. The gametophytes were later identified to species using the DNA barcodes.

### Statistical analysis

All statistical analyses were done in R v.`r getRversion()` [@RCoreTeam2020a].

We derived the phylogenetic tree used in this study by extracting Hymenophyllaceae from the tree of @Nitta2017 with the \function{extract.clade} function of the \package{ape} package [@Paradis2004].
This tree is in agreement with previously published, more densely sampled phylogenies [@Dubuisson2003; @Ebihara2007a; @Hennequin2006; @Pryer2001a].

We quantified the degree of phylogenetic signal in physiological traits using Blomberg's *K* [@Blomberg2003] and Pagel's λ [@Freckleton2002; @Pagel1999] as implemented with the \function{phylosig} function in the \package{phytools} package [@Revell2012].
Both measures test the hypothesis that the trait of interest is evolving according to Brownian Motion (BM).
λ is a scaling parameter that ranges from zero to one: λ near zero indicates random distribution of trait values across the tree, and λ near one indicates evolution of traits along the phylogeny following BM.
For *K*, values near one indicate evolution of traits following BM; *K* > 1 indicates traits more conserved than expected under BM, and *K* < 1 indicates that traits have less phylogenetic signal than expected under BM.
We tested the significance of *K* by comparing the observed value against values from a null distribution of 10,000 phylogenies with the traits randomly shuffled across the tips.
We tested the significance of λ with a log-likelihood test comparing the likelihood of the observed value of λ vs. λ = 0.
For all further analyses, we used methods that account for phylogenetic history for physiological traits that showed evidence of phylogenetic signal in at least one of the two metrics, and standard methods otherwise.

We used general linear mixed models (GLMMs) to investigate the effect of growth habit and life stage on DT (% recovery of F~v~/F~m~ after 48 h recovery from desiccation at -86 MPa for 2 d) and photosynthetic parameters (ETR~max~ and PPFD~95%~) using the \package{MCMCglmm} package [@Hadfield2010].
For each of the three response variables (species mean DT, ETR~max~, and PPFD~95%~), we constructed five models: a null model with no fixed effects, or models with either growth habit (high-elevation epiphyte, low-elevation epiphyte, terrestrial or saxicolous), generation (sporophyte vs. gametophyte), growth habit and generation, or growth habit, generation, and their interaction as fixed effects.
We included species as a random effect in all models, and included a phylogenetic variance-covariance matrix if the response variable showed evidence of phylogenetic signal [@Hadfield2010].
We used an inverse-Gamma distribution for all priors, and ran analyses for 500,000 iterations, with burnin after 1,000 iterations and thinning every 500 iterations.
We compared models using the Deviance Information Criterion (DIC) [@Spiegelhalter2002] and selected the best-fitting model as the one with the lowest DIC.

Comparisons of DT, ETR~max~, and PPFD~95%~ between gametophytes and sporophytes of the same species do not involve phylogeny, so we used a two-sided *t*-test for these.

To test for a correlation between environmental water availability and DT, we first derived a single maximum VPD for each species by calculating the average of the maximum daily VPD at all sites where that species was present.
We used maximum VPD (\ie, minimum humidity) values because these may represent a climatic extreme that limits the occurrence of species.
We used data from the 'epiphytic' dataloggers mounted at 1.5 m on trees for epiphytic species and data from the 'terrestrial' dataloggers on the ground for non-epiphytic species.
Data from the Mt. Rotui 830 m site were excluded because this site is much more exposed than the other high elevation sites, and appears as a significant outlier in the microclimatic data [@Nitta2020].
Since DT showed significant phylogenetic signal (see Results), we used phylogenetic generalized least squares (PGLS) [@Freckleton2002] to test for a relationship between VPD and DT while taking phylogeny into account, using species' means.

To test whether gametophytes with broader ranges have higher DT, we calculated the extent of gametophyte range beyond sporophytes ("G beyond S") as the total elevational range (m) gametophytes were observed growing beyond sporophytes at either end of the elevational gradient. 
We conducted PGLS with DT as the response variable and G beyond S as the explanatory variable with the \function{pgls} function in the \package{caper} [@Orme2018].

Datasets used in this study are available on FigShare (https://doi.org/10.6084/m9.figshare.14184572). 
Code used to analyze the data and generate figures and this manuscript are available on GitHub (https://github.com/joelnitta/moorea_filmies).
A Docker image is available to run the code at https://hub.docker.com/r/joelnitta/moorea_filmies.
The \package{targets} package [@Landau2021] was used to automate the workflow.
The \package{ggplot2} [@Wickham2016], \package{ggtree}, [@Yu2017] and \package{patchwork} (https://github.com/thomasp85/patchwork) packages were used to generate the figures.

\*\*\* NOTE TO READER: The DOI for the dataset will not be active until the paper is accepted. Until then, this link can be used for the dataset https://figshare.com/s/d6349abf01a3756a5aae  \*\*\*

## Results

### Field survey

In total, `r n_distinct(range_comparison$species)` filmy fern taxa (including two species that were recognized as multiple varieties) were observed on Moorea.
Three taxa with distinct *rbcL* sequences, morphology, and elevational ranges could be distinguished within the *Crepidomanes minutum* (Blume) K. Iwats. species complex, but species status of these taxa remains uncertain [@Nitta2011].
We refer to these here as *C. minutum* var. 1, 2, and 3.
Similarly, two members of the *Abrodictyum asae-grayi* (Bosch) Ebihara & K. Iwats. complex (also including *Abrodictyum caudatum* (Brack.) Ebihara & K. Iwats. in our tree; `r s_figure("tree")`) were observed on the basis of *rbcL* and morphology, but as taxonomic treatment is beyond the scope of the current study we refer to them as *A. asae-grayi* var. 1 and 2.

In several species, gametophytes exceeded the maximum or minimum elevational range of sporophytes including *C. apiifolia*, *C. minutum* var. 2 and 3, *Hymenophyllum digitatum* (Sw.) Fosberg, *Hymenophyllum javanicum* Spreng., *Hymenophyllum pallidum* (Blume) Ebihara & K.
Iwats., and *Polyphlebium borbonicum* (Bosch) Ebihara & Dubuisson (`r s_figure("range")`).
All of these have sporophytes that are normally restricted to cloud forest (\circa{ }500--1200 m), but gametophytes that are distributed to lower elevations.

Sporophytes of two taxa, *C. minutum* var. 3 and *Hymenophyllum braithwaitei* Ebihara & K. Iwats., did not occur in the study plots, but each was observed from a single population on Mt. Tohiea, at \circa{ }1000 m and near the summit (1206 m), respectively.
We include these observations for comparison of ranges between sporophytes and gametophytes.

### Phylogenetic signal

```{r phy-sig-stats}
sporo_recover_k <- phylosig %>% filter(generation == "sporophyte", trait == "recovery_mean") %>% pull(kval) %>% round_t(2)
sporo_recover_k_pval <- phylosig %>% filter(generation == "sporophyte", trait == "recovery_mean") %>% pull(k_pval) %>% round_t(3)
sporo_recover_lambda <- phylosig %>% filter(generation == "sporophyte", trait == "recovery_mean") %>% pull(lambda) %>% round_t(2)
sporo_recover_lambda_pval <- phylosig %>% filter(generation == "sporophyte", trait == "recovery_mean") %>% pull(lambda_pval) %>% round_t(3)

gameto_recover_k <- phylosig %>% filter(generation == "gametophyte", trait == "recovery_mean") %>% pull(kval) %>% round_t(2)
gameto_recover_k_pval <- phylosig %>% filter(generation == "gametophyte", trait == "recovery_mean") %>% pull(k_pval) %>% round_t(3)
gameto_recover_lambda <- phylosig %>% filter(generation == "gametophyte", trait == "recovery_mean") %>% pull(lambda) %>% round_t(2)
gameto_recover_lambda_pval <- phylosig %>% filter(generation == "gametophyte", trait == "recovery_mean") %>% pull(lambda_pval) %>% round_t(3)

# Make sure that all ETRmax and PAR have pval > 0.05
phylosig %>% 
  filter(trait != "recovery_mean") %>%
  verify(all(k_pval > 0.05)) %>%
  verify(all(lambda_pval > 0.05), success_fun = success_logical, error_fun = error_stop)
```

For DT, *K* was near 1 with *P* < 0.1 in both generations (gametophytes *K* = `r gameto_recover_k`, *P* = `r gameto_recover_k_pval`; sporophytes *K* = `r sporo_recover_k`, *P* = `r sporo_recover_k_pval`); λ was near 1 for both generations, but not significant for sporophytes (gametophytes λ = `r gameto_recover_lambda`, *P* = `r gameto_recover_lambda_pval`; sporophytes λ = `r sporo_recover_lambda`, *P* = `r sporo_recover_lambda_pval`). There was no evidence of significant phylogenetic signal in photosynthetic parameters (PPFD~95%~ and ETR~max~) in either generation (*P* >> 0.05, `r table("physig")`). Therefore, phylogenetic methods were used for analyses involving DT, but not photosynthetic parameters.

### Desiccation tolerance

```{r dt-stats}
# Count number of species included in DT test by generation
n_dt_species_sporo <- combined_species_means %>% 
  filter(!is.na(recovery_mean)) %>% 
  count(species, generation) %>% 
  count(generation) %>% 
  filter(generation == "sporophyte") %>% 
  pull(n) %>% 
  english2

n_dt_species_gameto <- combined_species_means %>% 
  filter(!is.na(recovery_mean)) %>%
  count(species, generation) %>% 
  count(generation) %>% 
  filter(generation == "gametophyte") %>% 
  pull(n) %>% 
  english2

n_dt_species_both <- combined_species_means %>% 
  filter(!is.na(recovery_mean)) %>% 
  count(species, generation) %>% 
  group_by(species) %>% 
  summarize(n_generation = n_distinct(generation), .groups = "drop") %>%
  filter(n_generation == 2) %>%
  pull(species) %>%
  n_distinct() %>%
  english2

# Extract effect of generation on DT from GLMM
sporo_dt_effect_mean <-
glmm_params %>% 
  filter(response == "recovery", predictor == "generationsporophyte") %>%
  verify(pMCMC < 0.05) %>%
  pull(post.mean) %>%
  scales::percent(accuracy = 0.1)

sporo_dt_effect_lower <-
glmm_params %>% 
  filter(response == "recovery", predictor == "generationsporophyte") %>%
  verify(pMCMC < 0.05) %>%
  pull(`l-95% CI`) %>%
  scales::percent(accuracy = 0.1)

sporo_dt_effect_upper <-
glmm_params %>% 
  filter(response == "recovery", predictor == "generationsporophyte") %>%
  verify(pMCMC < 0.05) %>%
  pull(`u-95% CI`) %>%
  scales::percent(accuracy = 0.1)

# Extract list of species with significant p-values for gameto DT less than sporo
dt_g_less_than_s <-
t_test_results %>% 
  filter(est_diff_sporo_gameto > 0, response == "recovery", p_value < 0.05) %>%
  abbrev_sp() %>%
  mutate(species = str_replace_all(species, "_", ". ")) %>%
  pull(species) %>%
  knitr::combine_words(before = "*")

# Extract list of species with significant p-values for gameto DT more than sporo
dt_g_more_than_s <-
t_test_results %>% 
  filter(est_diff_sporo_gameto < 0, response == "recovery", p_value < 0.05) %>%
  abbrev_sp() %>%
  mutate(species = str_replace_all(species, "_", ". ")) %>%
  pull(species) %>%
  knitr::combine_words(before = "*")

# Extract p-values for species with gameto DT less than sporo
dt_g_less_than_s_pvals <-
t_test_results %>%
  filter(est_diff_sporo_gameto > 0, response == "recovery", p_value < 0.05) %>%
  pull(species) %>%
  map_chr(~ttest_pval(species_select = ., response_select = "recovery", data = t_test_results)) %>%
  knitr::combine_words()

# Extract p-values for species with gameto DT more than sporo
dt_g_more_than_s_pvals <-
t_test_results %>%
  filter(est_diff_sporo_gameto < 0, response == "recovery", p_value < 0.05) %>%
  pull(species) %>%
  map_chr(~ttest_pval(species_select = ., response_select = "recovery", data = t_test_results)) %>%
  knitr::combine_words()

# Rel. water content
rel_water_table <- 
rel_water_species_means %>%
  filter(!is.na(rel_water)) %>%
  group_by(salt, dry_time, rec_time) %>% 
  summarize(mean = mean(rel_water), sd = sd(rel_water), n = n(), .groups = "drop") %>%
  arrange(mean) %>%
  mutate(across(c(mean, sd), ~scales::percent(., suffix = " %", accuracy = 0.1))) %>%
  mutate(sum_val = glue("{mean} ± {sd} (*n* = {n} species)"))

# Gameto DT
mean_gameto_dt_by_habit <-
combined_species_means %>%
  filter(!is.na(recovery_mean), generation == "gametophyte") %>%
  left_join(filmy_habit, by = "species") %>%
  arrange(recovery_mean) %>%
  group_by(habit) %>%
  summarize(
    mean = mean(recovery_mean),
    sd = sd(recovery_mean),
    n = n(),
    .groups = "drop"
  ) %>%
  mutate(across(c(mean, sd), ~scales::percent(., suffix = " %", accuracy = 0.1))) %>%
   mutate(sum_val = glue("{mean} ± {sd}, *n* = {n} species"))

gameto_dt_ter_mean <- mean_gameto_dt_by_habit %>%
  filter(habit == "Terrestrial") %>%
  pull(sum_val)
```

Ability to recover from desiccation was tested in sporophytes of `r n_dt_species_sporo` species and gametophytes of `r n_dt_species_gameto` species, including `r n_dt_species_both` species with both life phases.

A general correlation of DT with habitat was observed in sporophytes (`r figure("dt")`a).
The two terrestrial species lacked DT, although *C. apiifolia* showed slightly greater ability to recover by 48 h compared to *Abrodictyum dentatum* (Bosch) Ebihara & K. Iwats., which completely failed to recover from even the gentlest treatment of 2 d at --38 NaCl (more stringent tests were not conducted on *A. dentatum* after confirming its lack of DT at this level).
Saxicolous species (*Crepidomanes kurzii* (Bedd.) Tagawa & K. Iwats., *Polyphlebium endlicherianum* (C. Presl) Ebihara & K. Iwats., and *Vandenboschia maxima* (Blume) Copel.) also had low DT; only *C. kurzii* was able to recover from desiccation up to --86 MPa for 2 d but not more.
Nearly all epiphytic species were capable of recovering from desiccation of up to 2 d at --86 MPa.
*Crepidomanes bipunctatum*, a widespread low-elevation epiphyte, showed the greatest ability to recover from desiccation across all treatments.
Interestingly, high-elevation epiphytes *Hymenophyllum multifidum* (G. Forst.) Sw. and *Hymenophyllum polyanthos* (Sw.) Sw. could recover from desiccation at --86 or --282 MPa for 15 d, but not --38 MPa.

Gametophytes of terrestrial species could not tolerate desiccation (mean 48 h recovery `r gameto_dt_ter_mean`). Levels of DT were in general higher for saxicolous and epiphytic gametophytes, but otherwise did not show a clear pattern (`r figure("dt")`b).

Desiccation tolerance of gametophytes tended to be lower than sporophytes (`r figure("combined")`), and was significantly lower in `r dt_g_less_than_s` (*t*-test, *P* = `r dt_g_less_than_s_pvals`, respectively).
Only `r dt_g_more_than_s` showed significantly higher DT in gametophytes compared to sporophytes (*t*-test, *P* = `r dt_g_more_than_s_pvals`).

Control samples generally did not show any decrease in F~v~/F~m~ throughout the duration of the experiment (`r s_figure("dt-control")`) and maintained RWC near 100% (\figs{ }`r s_figure_num("water-all")`, `r s_figure_num("water-s")`). 
Relative water content in the desiccated state ranged from `r rel_water_table %>% filter(salt == "LiCl", dry_time == "15", rec_time == "desiccated") %>% pull(sum_val)` in the --282 MPa, 15 d treatment to `r rel_water_table %>% filter(salt == "NaCl", dry_time == "2", rec_time == "desiccated") %>% pull(sum_val)` in the --38 MPa, 2 d treatment (`r s_figure("water-all")`), and returned to \circa{ }100 % upon re-wetting in all species sampled (\figs{ }`r s_figure_num("water-all")`, `r s_figure_num("water-s")`).

Relative humidity in the growth chambers was high when samples were first inserted, then gradually decreased to the target value maintained by each drying salt (`r s_figure("chamber")`).
In a few cases (particularly species with larger fronds, \eg, *C. apiifolia*), the NaCl chamber did not reach equilibrium humidity before 2 d (`r s_figure("chamber")`).
Brief fluctuations in humidity were observed when chambers were opened to insert or remove samples (`r s_figure("chamber")`).

The GLMM including the combined effects of generation plus growth habit and their interaction was selected as the best-fitting model for DT (`r table("glmm")`). 
Sporophytes had `r sporo_dt_effect_mean` greater DT (95 % CI `r sporo_dt_effect_lower` to `r sporo_dt_effect_upper`) relative to gametophytes (`r table("glmm-pars")`).

### Photosynthetic optima

```{r photosyn-stats}
# Count number of species included in light data by generation
n_light_species_sporo <- combined_species_means %>% 
  filter(!is.na(etr_mean), !is.na(par_mean)) %>%
  count(species, generation) %>% 
  count(generation) %>% 
  filter(generation == "sporophyte") %>% 
  pull(n) %>% 
  english2

n_light_species_gameto <- combined_species_means %>% 
  filter(!is.na(etr_mean), !is.na(par_mean)) %>%
  count(species, generation) %>% 
  count(generation) %>% 
  filter(generation == "gametophyte") %>% 
  pull(n) %>% 
  english2

n_light_species_both <- combined_species_means %>% 
  filter(!is.na(etr_mean), !is.na(par_mean)) %>% 
  count(species, generation) %>% 
  group_by(species) %>% 
  summarize(n_generation = n_distinct(generation), .groups = "drop") %>%
  filter(n_generation == 2) %>%
  pull(species) %>%
  n_distinct() %>%
  english2

# Calculate grand means by generation for light responses
# (so mean is the mean of species means, and sd is sd of species means)
par_means <-
combined_species_means %>%
  filter(!is.na(par_mean)) %>%
  group_by(generation) %>%
  summarize(
    mean = mean(par_mean),
    sd = sd(par_mean),
    n = n(),
    .groups = "drop"
  ) %>%
  mutate(
    mutate(across(c(mean, sd), ~round_t(.x, 1))),
    sum_val = glue("{mean} ± {sd}"),
  )

etr_means <-
combined_species_means %>%
  filter(!is.na(etr_mean)) %>%
  group_by(generation) %>%
  summarize(
    mean = mean(etr_mean),
    sd = sd(etr_mean),
    n = n(),
    .groups = "drop"
  ) %>%
  mutate(
    mutate(across(c(mean, sd), ~round_t(.x, 1))),
    sum_val = glue("{mean} ± {sd}"),
  )

# Extract values by response and generation for citing in MS
mean_par_sporo <- par_means %>% filter(generation == "sporophyte") %>% pull(sum_val)
mean_par_gameto <- par_means %>% filter(generation == "gametophyte") %>% pull(sum_val)
mean_etr_sporo <- etr_means %>% filter(generation == "sporophyte") %>% pull(sum_val)
mean_etr_gameto <- etr_means %>% filter(generation == "gametophyte") %>% pull(sum_val)
mean_par_sporo_n <- par_means %>% filter(generation == "sporophyte") %>% pull(n)
mean_par_gameto_n <- par_means %>% filter(generation == "gametophyte") %>% pull(n)
mean_etr_sporo_n <- etr_means %>% filter(generation == "sporophyte") %>% pull(n)
mean_etr_gameto_n <- etr_means %>% filter(generation == "gametophyte") %>% pull(n)
```

Light responses were measured in sporophytes of `r n_light_species_sporo` species and gametophytes of `r n_light_species_gameto` species, including `r n_light_species_both` species with both life phases.

Photosynthetic optima showed clear differences across generations: sporophytes were adapted for higher light levels, whereas gametophytes were adapted for lower light levels (\figs{ }`r figure_num("combined")`, `r s_figure_num("light-curves")`, `r s_figure_num("traits-box")`).
Sporophytes reached their maximum photosynthetic rate (PPFD~95%~) at light levels ca.
`r mean_par_sporo` µmol·m^--2^ ·s^--1^ (*n* = `r mean_par_sporo_n` species), whereas gametophytes did so at `r mean_par_gameto` µmol·m^--2^ ·s^--1^ (*n* = `r mean_par_gameto_n` species).
Similarly, sporophytes attained higher rates of electron transport (ETR~max~) (`r mean_etr_sporo` µmol·m^--2^ ·s^--1^, *n* = `r mean_etr_sporo_n` species) than gametophytes (`r mean_etr_gameto` µmol·m^--2^ ·s^--1^, *n* = `r mean_etr_gameto_n` species).

The GLMM including the effect of generation only was selected as the best-fitting model for ETR~max~, and the combined effect of growth habit plus generation and their interaction was selected for PPFD~95%~ (`r table("glmm")`).

### Relationships of vapor pressure deficit and range size with DT

```{r pgls-stats}
sporo_vpd_pval <- env_range_dt_model_summary %>% filter(model == "sporo_vpd_model") %>% pull(p_value) %>%
  round_t(3)
gameto_vpd_pval <- env_range_dt_model_summary %>% filter(model == "gameto_vpd_model") %>% pull(p_value) %>%
  round_t(3)
```

Desiccation tolerance increased significantly with VPD for sporophytes, but not gametophytes (PGLS, *P* = `r sporo_vpd_pval`; `r figure("corr")`; `r table("pgls")`).

No significant relationship was detected between range size of gametophytes beyond sporophytes and DT (`r figure("corr")`, `r table("pgls")`).

## Discussion

Vegetative DT was a key innovation linked to the rise of early land plants, but is largely limited to non-vascular groups such as bryophytes and algae among extant plants.
Filmy ferns have received considerable attention for their remarkable reversion to vegetative DT in the sporophyte phase [\eg, @GarcesCea2014; @Ostria-Gallardo2020; @Proctor2003; @Shreve1911].
Yet, no previous study to our knowledge has evaluated parallel physiology of sporophytes and gametophytes of filmy ferns in relation to DT.
Here, we present the first comparisons of physiological parameters between generations within filmy ferns in a phylogenetic comparative framework.

### Filmy fern sporophytes have a wide range of DT

```{r sporo-dt-disc-stats}
# Calculate the avg daily maximum VPD tolerated by sax & terr sporophytes
max_ter_vpd <- mean_vpd_sporo %>%
  left_join(filmy_habit, by = "species") %>%
  filter(habit %in% c("Saxicolous", "Terrestrial")) %>%
  slice_max(order_by = vpd_sporophyte_mean) %>%
  pull(vpd_sporophyte_mean) %>%
  round(1)
```

We identified a wide variation of DT in filmy fern sporophytes on Moorea, from marked sensitivity in terrestrial species to tolerance of water potentials well below -200 MPa (VPD > 2.45 kPa) in some epiphytes.
Levels of similarly extreme DT have also been reported in other species of *Hymenophyllum* [@Proctor2003; @Proctor2012] and a general correlation of habitat with DT in filmy ferns has been previously established [@Parra2009; @Proctor2003; @Proctor2012; @Saldana2013; @Shreve1911], but no other studies to our knowledge have documented a similar range of DT values in filmy ferns from a single site.
One may wonder then, what is the adaptive significance of being able to withstand drying at -282 MPa (\circa{ }2.45 kPa) vs. -86 MPa (\circa{ }1.25 kPa) when the ambient VPD is typically < \circa{ }0.10 kPa and only rarely exceeds > 0.50 kPa.
The recent finding of a constitutive mechanism for DT in filmy ferns [@GarcesCea2014] supports the need for a relatively wide "safety margin," and similar (or even more extreme) levels of constitutive DT are well known in bryophytes [@Oliver1993].
The conservative behavior of fern stomata, which lack a response to the plant stress hormone ABA, is well documented [@Brodribb2011; @Mcadam2012; @Mcadam2013].
It is possible that filmy ferns also behave conservatively with respect to water-stress.
We suggest that the variation in DT observed here is relevant because it is correlated with habitat, and likely reflects overall physiological tolerance, not just artificial extremes.
For example, sporophytes of some high elevation epiphytes that we surveyed (*H. multifidum*, *H. polyanthos*) seem to be able to better withstand intense vs. moderate desiccation, with higher recovery values at -282 and -86 MPa vs. -38 MPa when dry for 15 d.
This may be an adaptation to cloud forest conditions; although cloud forests are frequently moist due to constant fog, dry periods are intense during the times when there is no cloud cover (J. Nitta, pers. obs.).
The complete lack of DT in terrestrial species is not surprising, considering they have much more direct access to water than epiphytes and therefore are under substantially less selective pressure to maintain DT.
Because controls kept at 100% RH maintained F~v~/F~m~ values near pre-treatment levels in all sporophytes (`r s_figure("dt-control")`), it is unlikely that the lack of tolerance seen in terrestrial species is solely due to removal from the soil.
The significant relationship between average daily maximum VPD and DT in sporophytes supports the specialization of sporophyte DT according to niche, and indicates that sporophytes of non-tolerant terrestrial and saxicolous species may be unable to occur in habitats with VPD exceeding \circa{ }`r max_ter_vpd` kPa (`r figure("corr")`).

### Filmy fern gametophytes are not more stress-tolerant than sporophytes

Contrary to our hypotheses, gametophytes of filmy ferns are in general not more tolerant of the abiotic environment (light levels and water potential) than sporophytes.
This differs from other fern groups with desiccation tolerant gametophytes but desiccation avoiding sporophytes [@WatkinsJr.2007c].
In the filmy ferns we studied, DT levels of both sporophytes and gametophytes largely reflected habitat; in general, epiphytic species tended to have higher DT than terrestrial species, regardless of life stage (\figs{ }`r figure_num("combined")`, `r s_figure_num("traits-box")`).
Interestingly, this echoes much earlier field observations (not experimental data) by @Holloway1930 on filmy fern gametophytes in New Zealand: he reported apparent DT in gametophytes of *Hymenophyllum rarum* R. Br. and *H. villosum* Colenso, both high elevation epiphytes with desiccation tolerant sporophytes, and lack of DT in terrestrial filamentous gametophytes of *Polyphlebium colensoi* (Hook. f.) Ebihara & K. Iwats., *Abrodictyum strictum* (Menzies ex Hook. & Grev.) Ebihara & K. Iwats., and *A. elongatum* (A. Cunn.) Ebihara & K. Iwats. occupying moist, protected sites on the forest floor.
@WatkinsJr.2007c also found that DT of fern gametophytes tended to match habitat, with terrestrial species having lower DT than epiphytes.
However, we found that gametophytes had lower DT than sporophytes regardless of growth habit (\figs{ }`r figure_num("combined")`, `r s_figure_num("traits-box")`).
At high elevation sites (\ie, cloud forest), epiphytic ferns often occur in a dense matrix of bryophytes (`r figure("habit")`b).
It is possible that this bryophyte cover helps to retain moisture during brief periods of drought and thus protect the epiphytic fern gametophytes that grow amongst them [@Harrington2021].
More detailed studies of the high elevation epiphytic environment and interactions between co-occurring gametophytes of ferns and bryophytes are required to test this hypothesis.

Although we predicted that species with gametophytes that occur far beyond the range of their sporophytes would have higher DT, this was not the case (`r figure("corr")`c).
This is perhaps best illustrated by *C. apiifolia* (`r figure("habit")`d, e, h).
Sporophytes of this species are usually restricted to moist cloud forest sites from \circa{ }600 m to 1200 m, but we observed gametophytes over a much wider range, down to 200 m (`r s_figure("range")`).
A similar distribution pattern has been reported in congeneric *Callistopteris baldwinii* (D. C. Eaton) Copel.
from Hawaii [@Dassler1997].
Since gametophytes of *C. apiifolia* are common on Moorea and have a unique morphology [@Nitta2017], we were able to collect sufficient material of this species for additional tests of DT at 100% RH for 2 d.
However, *C. apiifolia* gametophytes still failed to recover from even this gentle treatment (`r s_figure("capi-water")`).
This suggests that the widespread gametophytes of *C. apiifolia* do not rely on DT to attain their distribution, but instead must do so via some other mechanism.
Although we measured terrestrial microhabitats by positioning sensors a few cm above the ground, these sensors are still outside of the boundary layer.
It is likely that the gametophytes of *C. apiifolia* do not actually experience the conditions measured with our 'terrestrial' dataloggers, but instead grow within a more consistently moist boundary layer.
Gametophytes of *C. apiifolia* produce gemmae (`r figure("habit")`e, h), and often form dense clonal mats.
In the case that their microhabitats do become too dry, they may suffer loss at the edge of the population, then recover by clonal growth [@Farrar2008].
Furthermore, the complex surface of clonal mats has been shown to increase the boundary layer and decrease evaporation rates in mosses [@Rice2001], and may function similarly in *C. apiifolia*.
Other filmy fern species also produce gemmae, but they are much more frequent in *C. apiifolia*, and the large size of the clonal mats formed by *C. apiifolia* indicate that it may have a high growth rate (J. Nitta, pers. obs.).
It is possible that there is a tradeoff between growth rate and DT, such that species capable of rapid asexual growth like *C. apiifolia* are less able to tolerate desiccation, but slower growing epiphytes such as *Hymenophyllum* spp.
can better tolerate it [@Oliver2000].
Growth rates of tolerant versus non-tolerant gametophytes, including quantification of gemmae production, should be investigated in these species to test this hypothesis.

### Gametophytes are adapted for lower light levels than sporophytes

We observed a clear intergenerational difference in photosynthetic optima, with gametophytes consistently adapted for lower light levels than sporophytes (`r figure("combined")`, `r table("glmm")`).
In the only other study to our knowledge that compared photosynthetic rates between gametophytes and sporophytes of a filmy fern, @Johnson2000 also found that gametophytes of *V. speciosa* reached maximum ETR at lower levels than sporophytes: in gametophytes of this species, photosynthesis became saturated at a PPFD of 4--5 µmol·m^-2^·s^-1^ in the field or 30--50 µmol·m^-2^·s^-1^ in the lab vs. 30--50 µmol·m^-2^ ·s^-1^ for sporophytes (lab measurement only).
Although we found similar differences in direction between gametophytes and sporophytes, the measurements of @Johnson2000 are considerably lower than what we observed in the field for both Moorean filmy fern gametophytes (PPFD~95%~ \circa{ }40--60 µmol·m^-2^·s^-1^) and sporophytes (PPFD~95%~ \circa{ }130--160 µmol·m^-2^·s^-1^).
This makes sense in light of the fact that *V. speciosa* is an extremely deep-shade plant, with gametophytes occupying protected rock crevices that receive less than 1 µmol·m^-2^·s^-1^ PPFD for most of the day [@Johnson2000]; many of the gametophytes sampled in our study occur in more exposed conditions.
It is surprising, however, that we did not observe any consistent differences between species in different habitats, given that light levels should vary between tree trunks and the forest floor, at least on a \circa{ }1 m scale.
Although the GLMM showed an interaction effect between generation and growth habit for PPFD~95%~, the model including only generation was selected for ETR~max~ (`r table("glmm")`).
It is possible that the actual light levels experienced by gametophytes growing in these habitats may not vary significantly if they occupy protected microsites such as bryophyte mats or crevices in rock or bark.

### Influence of phylogeny on traits

Previous studies of filmy fern ecophysiology also found a link between habitat and DT and/or photosynthetic rates [@Parra2009; @Proctor2003; @Proctor2012; @Saldana2013]; however, ours is the first to our knowledge to investigate this pattern while taking phylogeny into account.
We detected phylogenetic signal in DT (`r table("physig")`), suggesting that phylogeny should be considered when interpreting differences in DT between species.
In contrast, the lack of phylogenetic signal in photosynthetic traits, coupled with the clear intergenerational differences we observed, suggest that sporophytes and gametophytes are adapted for different light levels regardless of clade or growth habit.
In a more densely sampled study of morphological evolution in the trichomanoid clade, @Dubuisson2013 identified several morphological changes including reduced stele and root systems associated with the evolution of epiphytic growth within the HE subclade (comprising the genera *Polyphlebium*, *Didymoglossum*, *Crepidomanes*, and *Vandenboschia*).
It is possible that both reduced morphology and increased DT are parts of an integrated adaptive strategy for epiphytic growth within this subclade of filmy ferns, and have evolved together repeatedly.
However, @Dubuisson2013 were unable to determine if these morphological changes had occurred independently, or if there was a single origin at the base of the HE clade followed by multiple losses.
Increased sampling of both morphological and physiological traits of additional trichomanoid taxa is needed to distinguish between these competing scenarios.

### Concluding remarks and future directions

In filmy ferns, both sporophyte and gametophyte generations are poikilohydric; however, this does not necessarily mean they share similar physiological optima.
Although filmy fern sporophytes are often of small stature, elevating a leaf even a few cm above the substrate surface means escape from the boundary layer and exposure to substantially greater evapotranspiration.
It is likely that even in this clade, wherein the physiological divide between sporophyte and gametophyte is probably the smallest amongst ferns, the two generations experience substantially different selective pressures.
Fossils indicate that some of the earliest vascular plants may have had sporophytes and gametophytes that were both branched and small [@Kenrick1997a], but it is unclear at what point selective pressures began to diverge to favor large, complex sporophytes and small, simple gametophytes.
The relevance of plant size to water relations is supported by a study showing increased DT in smaller size classes of the epiphytic fern *Asplenium auritum* Sw. [@Testo2012].
Future studies of DT in filmy ferns should include juvenile stages of the sporophyte, which are expected to experience similar microhabitats as gametophytes, to better determine the role of microhabitat vs. life stage (and plant size) on DT.
The recent application of transcriptomics to filmy fern ecophysiology has begun to reveal the genetic mechanisms underlying DT in these plants [@Ostria-Gallardo2020; @Ostria-Gallardo2020a].
Future studies that apply such methods to gametophytes as well as sporophytes across a variety of growth habits are anticipated to provide even greater insight into the ecology and evolution of filmy ferns as well as the transition from desiccation tolerance to avoidance in vascular plants.

**Acknowledgments**

Members of the Davis Lab and Jonathan Losos provided helpful discussion and comments on drafts.
Saad Amer, Ravahere Taputuarai, Tohei Theophilus, and Suzanne Vinnette assisted with fieldwork and experiments.
Weston Testo provided advice on settings for the DT test.
Staff at the University of California Berkeley's Richard B. Gump South Pacific Research Station, Moorea, French Polynesia, in particular Valentine Brotherson, Neil Davies, Hinano Teavai-Murphy, and Frank Murphy, provided logistical support for fieldwork.
Funding provided in part by the National Science Foundation (Doctoral Dissertation Improvement Grant DEB-1311169 to JHN and CCD), Setup Funds from Harvard University to CCD, American Society of Plant Taxonomists (Research Grant for Graduate Students to JHN), Garden Club of America (Award in Tropical Botany to JHN), Harvard University Herbaria (Fernald Fieldwork Fellowship to JHN), Society of Systematic Biologists (Graduate Student Research Award to JHN), and Systematics Association (Systematics Research Fund to JHN).

**Electronic supplementary material**

The online version of this article (DOI) contains supplementary material, which is available to authorized users.

## References

<!-- Insert <div id="refs"></div> to place the refs here instead of at the end -->
<div id="refs"></div>

\newpage

## Tables

```{r table-setup, purl = FALSE}
options(knitr.table.format = "latex")
```

**`r table("physig")`** Results of tests for phylogenetic signal (Blomberg’s *K* and Pagel’s λ) in desiccation tolerance (DT) and photosynthetic parameters of filmy ferns from Moorea, French Polynesia by generation

```{r physig-table, results = "asis", purl = FALSE}
# Format phylogenetic signal results as a pretty table with kableExtra

# Make helper table for formatting by groups
phylosig_group_locs <-
phylosig %>% mutate(row_num = 1:nrow(.)) %>% group_by(generation) %>%
  summarize(
    start = min(row_num),
    end = max(row_num),
    .groups = "drop"
  )

phylosig %>%
  mutate(
    kval = ifelse(kval < 0.001, format(kval, digits = 3), round_t(kval, 3)),
    lambda = ifelse(lambda < 0.001, format(lambda, digits = 3), round_t(lambda, 3)),
    k_pval = ifelse(k_pval < 0.001, format(k_pval, digits = 3), round_t(k_pval, 3)),
    lambda_pval = ifelse(lambda_pval < 0.001, format(lambda_pval, digits = 3), round_t(lambda_pval, 3)),
    trait = str_replace_all(trait, "recovery_mean", "Recovery") %>%
      str_replace_all("etr_mean", "ETR\\\\textsubscript{max}") %>%
      str_replace_all("par_mean", "PPFD\\\\textsubscript{95\\\\%}"),
    generation = str_to_sentence(generation)
  ) %>%
  select(trait, kval, k_pval, lambda, lambda_pval) %>%
  kbl(format = "latex", booktabs = TRUE, col.names = c("", "\\textit{K}","\\textit{P}(\\textit{K})", "λ","\\textit{P}(λ)"), escape = FALSE) %>%
  pack_rows(
    "Sporophytes", 
    phylosig_group_locs %>% filter(generation == "sporophyte") %>% pull(start),
    phylosig_group_locs %>% filter(generation == "sporophyte") %>% pull(end),
    escape = FALSE) %>%
  pack_rows(
    "Gametophytes", 
    phylosig_group_locs %>% filter(generation == "gametophyte") %>% pull(start),
    phylosig_group_locs %>% filter(generation == "gametophyte") %>% pull(end),
    escape = FALSE)
```

Recovery, recovery of F~v~/F~m~ (%) following 2 d desiccation at –86 MPa; ETR~max~, maximum relative electron transport rate; PPFD~95%~, light level at which 95% of ETR~max~ is reached.

\newpage

**`r table("glmm")`** Deviance information criterion (DIC) and delta DIC for general linear mixed models (GLMMs) of photosynthetic parameters and desiccation tolerance in response to generation and growth habit in filmy ferns from Moorea, French Polynesia

```{r glmm-table, results = "asis", purl = FALSE}
# Make helper table for formatting by groups
glmm_summ_group_locs <-
glmm_summary %>% mutate(row_num = 1:nrow(.)) %>% group_by(response) %>%
  summarize(
    start = min(row_num),
    end = max(row_num),
    .groups = "drop"
  )

# Format column names.
# left side is new name (with latex formatting)
# right side is original name
glmm_summary_cols <-
  c(
    "Fixed effects" = "fixed_effects", 
    "DIC" = "DIC",
    "delta DIC" = "delta_DIC"
  )

glmm_summary %>%
  mutate(
    fixed_effects = case_when(
      fixed_effects == "both" ~ "Generation + habit",
      fixed_effects == "intersect" ~ "Generation + habit + Generation:habit",
      TRUE ~ str_to_sentence(fixed_effects)
    ),
    DIC = round_t(DIC, 2),
    delta_DIC = round_t(delta_DIC, 2)
  ) %>%
  select(all_of(glmm_summary_cols)) %>%
  kbl(format = "latex", booktabs = TRUE, escape = TRUE) %>%
  row_spec(which(glmm_summary$delta_DIC == 0), bold = TRUE) %>%
  pack_rows(
    "ETR\\\\textsubscript{max}", 
    glmm_summ_group_locs %>% filter(response == "etr") %>% pull(start),
    glmm_summ_group_locs %>% filter(response == "etr") %>% pull(end),
    escape = FALSE) %>%
  pack_rows(
    "PPFD\\\\textsubscript{95\\\\%}", 
    glmm_summ_group_locs %>% filter(response == "par") %>% pull(start),
    glmm_summ_group_locs %>% filter(response == "par") %>% pull(end),
    escape = FALSE) %>%
  pack_rows(
    "Recovery", 
    glmm_summ_group_locs %>% filter(response == "recovery") %>% pull(start),
    glmm_summ_group_locs %>% filter(response == "recovery") %>% pull(end),
    escape = FALSE)
```

ETR~max~, maximum relative electron transport rate; PPFD~95%~, light level at which 95% of ETR~max~ is reached; Recovery, recovery of F~v~/F~m~ (%) following 2 d desiccation at –86 MPa

Best model (that with the lowest DIC) for each response variable in bold

\newpage

**`r table("glmm-pars")`** Predictor effects, lower and upper 95% confidence interval (CI), effective sample size, and significance of model parameters for the best general linear mixed models (GLMMs) of photosynthetic parameters and desiccation tolerance in response to generation and growth habit in filmy ferns from Moorea, French Polynesia

```{r glmm-params, results = "asis", purl = FALSE}
# Make helper table for formatting by groups
glmm_params_group_locs <-
glmm_params %>% mutate(row_num = 1:nrow(.)) %>% group_by(response) %>%
  summarize(
    start = min(row_num),
    end = max(row_num),
    .groups = "drop"
  )

# Format column names.
# left side is new name (with latex formatting)
# right side is original name
glmm_params_cols <-
  c(
    "Fixed effects" = "fixed_effects", 
    "Predictor" = "predictor",
    "Mean effect" = "post.mean",
    "Lower 95\\% CI" = "l-95% CI",
    "Upper 95\\% CI" = "u-95% CI",
    "Ef. sample size" = "eff.samp",
    "\\textit{P} MCMC" = "pMCMC"
  )

# Format cell values
glmm_params_table <-
glmm_params %>%
  mutate(
    fixed_effects = case_when(
      fixed_effects == "both" ~ "Generation + habit",
      fixed_effects == "intersect" ~ "Generation + habit + Generation:habit",
      TRUE ~ str_to_sentence(fixed_effects)
    ),
    predictor = str_replace_all(predictor, "generationsporophyte", "Sporophyte") %>%
      str_replace_all("habitLow_Epiphyte", "Low Epiphyte") %>%
      str_replace_all("habitSaxicolous", "Saxicolous") %>%
      str_replace_all("habitTerrestrial", "Terrestrial"),
    # Round off long numeric values
    across(c(post.mean, `l-95% CI`, `u-95% CI`, pMCMC), ~round_t(., 3)),
    eff.samp = as.integer(eff.samp)
  ) %>%
  # Format column names
  select(all_of(glmm_params_cols)) %>%
  kbl(format = "latex", booktabs = TRUE, escape = FALSE) %>%
  # Make significant rows bold
  row_spec(which(glmm_params$pMCMC < 0.05), bold = TRUE) %>%
  # Add group rows
  pack_rows(
    "ETR\\\\textsubscript{max}", 
    glmm_params_group_locs %>% filter(response == "etr") %>% pull(start),
    glmm_params_group_locs %>% filter(response == "etr") %>% pull(end),
    escape = FALSE) %>%
  pack_rows(
    "PPFD\\\\textsubscript{95\\\\%}", 
    glmm_params_group_locs %>% filter(response == "par") %>% pull(start),
    glmm_params_group_locs %>% filter(response == "par") %>% pull(end),
    escape = FALSE) %>%
  pack_rows(
    "Recovery", 
    glmm_params_group_locs %>% filter(response == "recovery") %>% pull(start),
    glmm_params_group_locs %>% filter(response == "recovery") %>% pull(end),
    escape = FALSE)

if(params$output_type == "preprint") glmm_params_table <- kable_styling(glmm_params_table, latex_options = "scale_down")

  glmm_params_table
  
```

ETR~max~, maximum relative electron transport rate; PPFD~95%~, light level at which 95% of ETR~max~ is reached; Recovery, recovery of F~v~/F~m~ (%) following 2 d desiccation at –86 MPa

Effect of generation is relative to gametophytes; effect of growth habit is relative to high elevation epiphytes

Significant predictors in bold

\newpage

**`r table("pgls")`** Phylogenetic generalized least squares (PLGS) models of desiccation tolerance as a function of vapor pressure deficit (VPD) or the range of gametophytes beyond sporophytes (G beyond S) in filmy ferns from Moorea, French Polynesia

```{r pgls, results = "asis", purl = FALSE}
# Format column names.
# left side is new name (with latex formatting)
# right side is original name
pgls_cols <-
  c(
    "Generation" = "generation", 
    "Predictor" = "predictor",
    "df" = "df",
    "\\textit{F}" = "f_value",
    "adj. \\textit{R}\\textsuperscript{2}" = "adj_r_squared",
    "\\textit{P}" = "p_value"
  )

# Format cell values
env_range_dt_model_summary %>%
  mutate(generation = case_when(
    str_detect(model, "sporo") ~ "S",
    str_detect(model, "gameto") ~ "G"
  )) %>%
  assert(not_na, generation) %>%
  mutate(
    predictor = case_when(
      str_detect(model, "vpd") ~ "VPD",
      str_detect(model, "range") ~ "G beyond S"
    )
  ) %>%
  assert(not_na, predictor) %>%
  mutate(
    f_value = round_t(f_value, 3),
    p_value = round_t(p_value, 3),
    adj_r_squared = round_t(adj_r_squared, 3)
  ) %>%
  # Format column names
  select(all_of(pgls_cols)) %>%
  kbl(format = "latex", escape = FALSE, booktabs = TRUE)
```

S, sporophyte; G, gametophyte

\newpage

## Figure legends

<!-- JPR requirements: Figure captions begin with the term Fig. in bold type, followed by the figure number, also in boldtype. No punctuation is to be included after the number, nor is any punctuation to be placed at the end of the caption. -->

**`r figure("habit")`** Examples of filmy ferns from French Polynesia.
(**a**–**d**) sporophytes; (**e**–**h**) gametophytes.
(**a**) *Crepidomanes humile*, a low elevation epiphyte. *Nitta 3392* (GH). 
(**b**) *Hymenophyllum multifidum*, a high elevation epiphyte growing in association with leafy liverworts and mosses. *Nitta 1257*
(GH). 
(**c**) *Polyphlebium endlicherianum*, a saxicolous species found at low to middle elevations. *Nitta 629* (GH). 
(**d**) *Callistopteris apiifolia*, a terrestrial species with sporophytes occurring only in humid cloud forests. *Nitta 3990* (GH).
(**e**) *C. apiifolia* (ribbon morphotype). Arrow indicates gemmae (asexual propagules) at tip of lobe. *Nitta 2565* (GH). 
(**f**) *Hymenophyllum polyanthos* (ribbon morphotype). *Nitta 3862* (GH).
(**g**) Filamentous gametophytes of an unidentified species of Hymenophyllaceae. *Nitta 4036* (GH). 
(**h**) *C. apiifolia*, showing detail of gemmifers (attachment points of gemmae, example indicated by arrow) and branched gemmae. *Nitta 3887* (GH). 
Scalebars: (**a**–**c**) 1.0 cm; (**d**) 5.0 cm; (**e**) 5.0 mm; (**f**) 2.0 mm; (**g**, **h**) 1.0 mm. 
Photographs by J. H. Nitta.

**`r figure("dt")`** Desiccation tolerance of filmy fern sporophytes (**a**) and gametophytes (**b**) from Moorea, French Polynesia.
Recovery was measured by comparing maximum photochemical yield of photosystem II before desiccation treatment with values at 30 min, 24 hr, and 48 hr following desiccation at three different intensities for either 2 or 15 days (see Methods).
Recovery was not measured for all combinations of species and desiccation treatments; missing points indicate data not collected (not recovery of 0%).
Error bars indicate standard deviation.
*n* = 8 samples per species/treatment unless otherwise indicated.
Growth habit indicated in parentheses after each species: T, terrestrial; S, saxicolous; LE, low elevation epiphyte; HE, high elevation epiphyte

**`r figure("combined")`** Phylogeny of filmy ferns from Moorea, French Polynesia with comparison of physiological parameters (mean ± 1 SD) between sporophytes (dark grey bars) and gametophytes (light grey bars).
Phylogeny adapted from Nitta et al. (2016); only species with physiological data shown.
Bootstrap support >50% from 100 ML bootstraps indicated at nodes.
Bold letters indicate major clades (H = hymenophylloid clade, T = trichomanoid clade).
Growth habit indicated by colored symbols (blue = low elevation epiphyte, green = high elevation epiphyte, yellow = saxicolous, red = terrestrial).
*Crepidomanes bipunctatum* and *C. humile* were observed growing as both epiphytes and saxicoles, but are shown as epiphytes to distinguish them from exclusively saxicolous species.
Asterisk indicates significant difference in means (*P* <0.05) between generations for a particular species (*t*-test); ns indicates no significant difference; lack of ns or asterisk means not enough observations available for that species for *t*-test.
Left bar plot shows recovery (%) of chlorophyll fluorescence (F~v~/F~m~) following 2 day desiccation at –86 MPa (*Abrodictyum dentatum* was desiccated at –38 MPa instead of –86 MPa). 
Middle bar plot shows maximum relative electron transport rate of photosystem II (ETR~max~) (µmol e-·m^-2^·s^-1^). 
Right bar plot shows photosynthetic photon flux density at 95% saturation of RETR (PPFD~95%~) (µmol quanta·m^-2^·s^-1^).
Absence of a bar indicates measurements not available for gametophytes or sporophytes of that species.

**`r figure("corr")`** Recovery of F~v~/F~m~ (%) following 2 d desiccation at –86 MPa plotted against environmental vapor pressure deficit (VPD; kPa) (**a**, **b**) or range of gametophytes beyond sporophytes (m) (**c**) for gametophytes (**a**, **c**) or sporophytes (**b**). 
Species means ± 1 SD are shown. 
Growth habit indicated by colored symbols (blue = low elevation epiphyte, green = high elevation epiphyte, yellow = saxicolous, red = terrestrial). 
*Crepidomanes bipunctatum* and *C. humile* were observed growing as both epiphytes and saxicoles, but were treated as epiphytes to distinguish them from exclusively saxicolous species.
*Abrodictyum dentatum* sporophytes were desiccated at –38 MPa instead of –86 MPa.
Range of gametophytes beyond sporophytes calculated as the total elevation that gametophytes exceeded sporophytes at either end of the elevational gradient.
Relationships between recovery and VPD or range of gametophytes beyond sporophytes were analyzed with phylogenetic generalized least squares (PGLS); significant relationships (*P* < 0.05) shown with grey line.

```{r plot-setup, purl = FALSE}
# JPR requirements:
# Figures should be 39 mm, 84 mm, 129 mm, or 174 mm wide and not higher than 234 mm
# In inches, widths: 1.53, 3.31, 5.08, 6.85; height: 9.21

# Set overall theme, saving default theme
default_theme <- theme_set(
  theme_bw(base_size = 10) +
    theme(
      panel.grid.minor = element_blank()
    ))

# Define color-blind palette
cbPalette <- c(grey = "#999999", gold = "#E69F00", skyblue = "#56B4E9", 
               green = "#009E73", yellow = "#F0E442", blue = "#0072B2", 
               red = "#D55E00", purple = "#CC79A7")
```

```{r dt-plot-setup, purl = FALSE}
### Format data for plotting ###

# Format labels for number of samples per species/generation
n_labels <-
recovery_species_means %>%
  group_by(species, generation) %>%
  mutate(n_count = n_distinct(recovery_n)) %>%
  mutate(
    n_label = case_when(
      # no label if exactly 8
      n_count == 1 & recovery_n == 8 ~ "",
      # single number if no. samples is NOT 8 and consistent across treatments
      n_count == 1 & recovery_n != 8 ~ as.character(glue(", *n* = {recovery_n}")),
      # range if no. samples is NOT 8 and varies across treatments
      n_count > 1 ~ as.character(glue(", *n* = {min(recovery_n)}-{max(recovery_n)}"))
    )) %>%
  select(species, generation, n_label) %>%
  unique() %>%
  ungroup()

dt_plot_data <-
  recovery_species_means %>%
  # Add growth habit data
  left_join(filmy_habit, by = "species") %>%
  assert(not_na, habit) %>%
  # Add labels for number of samples
  left_join(n_labels, by = c("species", "generation")) %>%
  # Abbreviate growth habit, set as factor and specify levels
  mutate(
    habit = case_when(
      habit == "High_Epiphyte" ~ "HE",
      habit == "Low_Epiphyte" ~ "LE",
      habit == "Saxicolous" ~ "S",
      habit == "Terrestrial" ~ "T"
    ) %>% factor(., levels = c("T", "S", "LE", "HE"))) %>%
  # Format labels:
  # - abbreviate species names to "G. species", make italic, add growth habit
  #   and sample number if not 8 
  mutate(
    label = str_replace_all(species, "[a-z]+_", ". "),
    label = glue("*{label}* ({habit}{n_label})")
  ) %>%
  # Add a line break only for C. bipunctatum
  mutate(label = str_replace_all(
    label, 
    fixed("*C. bipunctatum* (LE, *n* = 10-12)"),
    "*C. bipunctatum*<br>(LE, *n* = 10-12)")) %>%
  # - order by alphabetical species within growth habit
  arrange(habit, species) %>%
  mutate(
    order = 1:nrow(.),
    # fct_reorder must reorder based on a numeric vector
    label = fct_reorder(label, order)) %>%
  # Tweak variable names
  rename(recovery = recovery_mean, sd = recovery_sd, n = recovery_n)

### Set up plotting variables ###

# Set colors, shapes for desiccation intensity
salt_cols <- c(
  "NaCl" = cbPalette[["blue"]], 
  "MgNO3" = cbPalette[["yellow"]], 
  "LiCl" = cbPalette[["red"]])

salt_shapes <- c(
  "NaCl" = 21, # circle
  "MgNO3" = 24, # triangle
  "LiCl" = 23 # diamond
)

# Set offset of datapoints (dodge)
pd <- position_dodge(.3)

### Define function for making dt plot ###
make_dt_plot <- function (dt_plot_data, generation_select, plot_part) {
  dt_plot_data %>% filter(generation == generation_select) %>%
  ggplot(aes(x = rec_time, y = recovery, group = interaction(dry_time, salt), shape = salt, fill = salt)) +
  geom_line(position = pd, aes(linetype = dry_time)) +
  geom_errorbar(
    aes(ymin = recovery - sd, ymax = recovery + sd), 
    colour = "grey50", width = 0, 
    position = pd) +
  geom_point(position = pd, size = 1.5) +
  scale_linetype_manual(name = "Desiccation Time",
                        values = c("solid", "dashed"),
                        breaks = c("15", "2"),
                        labels = c("15 days", "2 days")) +
  scale_shape_manual(name = "Desiccation Intensity",
                     values = salt_shapes,
                     breaks = c("NaCl", "MgNO3",  "LiCl", "Control"),
                     labels = c("-38 MPa", "-86 MPa", "-282 Mpa", "Control")) + 
  scale_fill_manual(name = "Desiccation Intensity",
                    values = salt_cols, 
                    breaks = c("NaCl", "MgNO3",  "LiCl", "Control"),
                    labels = c("-38 MPa", "-86 MPa", "-282 Mpa", "Control")) +
  scale_y_continuous(label = label_percent(accuracy = 1), limits = c(-0.1,1.2), expand = c(0,0), breaks=c(0,.25,.5,.75,1.0)) +
  scale_x_discrete(limits = c("30min","24hr","48hr"),
                   labels = c("0.5", "24", "48")) +
  xlab("Recovery Time (hr)") +
  ylab("Recovery (%)") +
  labs(
    x = "Recovery Time (hr)",
    y = "Recovery",
    title = glue("**{plot_part}**")
  ) +
  theme(
    legend.position = "bottom",
    plot.title = element_markdown(),
    panel.grid.minor = element_blank(), 
    panel.grid.major.x = element_blank(),
    # Use markdown to format facet strip labels
    strip.text = element_markdown(),
    # Wrap legend parts into multiple lines
    legend.box = "vertical", 
    legend.margin = margin()
  ) + 
  facet_wrap(~label, ncol = 4)
}
```

<!-- From here, use params$output_type to output plots directly for preprint PDF, or save to file for journal submission -->

```{r filmies-print, fig.width = 6.85, results = "asis", fig.cap = figure("habit"), eval = params$output_type == "preprint", purl = FALSE}
# Print plot if knitting MS to preprint
knitr::include_graphics(here::here("images/figure_1.png"))
```

```{r filmy-dt-plot-make, purl = FALSE}
# Make plot
filmy_dt_plot <- make_dt_plot(dt_plot_data, "sporophyte", "a")
```

```{r filmy-dt-plot-save, eval = params$output_type == "ms", purl = FALSE}
# Save plot to file if knitting MS for submission
filmy_dt_plot_name <- figure("dt") %>% str_remove_all("[:punct:]| ") %>% paste0("a.pdf")
ggsave(plot = filmy_dt_plot, file = here::here("results/ms", filmy_dt_plot_name), width = 6.85, height = 7.5)
```

```{r filmy-dt-plot-print, fig.width = 6.85, fig.height = 7.5, fig.cap = figure("dt"), eval = params$output_type == "preprint", purl = FALSE}
# Print plot if knitting MS to preprint
filmy_dt_plot
```

```{r gameto-dt-plot-make, purl = FALSE}
# Make plot
gameto_dt_plot <- make_dt_plot(dt_plot_data, "gametophyte", "b")
```

```{r gameto-dt-plot-save, eval = params$output_type == "ms", purl = FALSE}
# Save plot to file if knitting MS for submission
gameto_dt_plot_name <- figure("dt") %>% str_remove_all("[:punct:]| ") %>% paste0("b.pdf")
ggsave(plot = gameto_dt_plot, file = here::here("results/ms", gameto_dt_plot_name), width = 6.85, height = 6.5, device = "pdf")
```

```{r gameo-dt-plot-print, fig.width = 6.85, fig.height = 7.5, fig.cap = paste(figure("dt"), "(continued)"), eval = params$output_type == "preprint", purl = FALSE}
# Print plot if knitting MS to preprint
gameto_dt_plot
```

```{r combined-plot-make, purl = FALSE}
# Make plot

# Copy data so they can be manipulated for this plot
phy <- filmy_phy
traits <- combined_species_means
habit <- filmy_habit

# Set colors for growth habit, generation
habit_colors <- c(
  "Terrestrial" = cbPalette[["red"]], 
  "Saxicolous" = cbPalette[["gold"]], 
  "Low_Epiphyte" = cbPalette[["green"]],
  "High_Epiphyte" = cbPalette[["blue"]])

generation_colors <- c(
  "gametophyte" = "grey70",
  "sporophyte" = "grey30"
)

# Subset tree to only species with trait data
phy <- drop.tip(phy, setdiff(phy$tip.label, traits$species))

# Sort species by position in phylogeny
traits <-
  traits %>%
  filter(species %in% phy$tip.label) %>%
  mutate(species = factor(species, levels = jntools::get_tips_in_ape_plot_order(phy)))

# Filter growth habit data to only those in phylogeny
habit <-
  habit %>%
  filter(species %in% phy$tip.label)
  
# Part A (left): tree
a <- ggtree(phy) %<+% habit +
  geom_tippoint(aes(color = habit), position = position_nudge(x = 14, y = 0), size = 3.0) +
  scale_color_manual(values = habit_colors, labels = pretty_text) +
  labs(color = "Growth habit") +
  # There is no way modify the text in geom_treescale (to add units), so make the
  # text disappear by setting fontsize = 0
  geom_treescale(y = 12, x = 2, width = 40, fontsize = 0) +
  # Manually place custom label above scale bar
  annotate("text", x = 22, y = 12, label = "40 my", size = 2.5, vjust = -1) +
  scale_x_continuous(expand = expansion(mult = c(0.01, 0.04))) +
  theme_void(base_size = 10)

# Part B (middle): desiccation tolerance
# Make helper table of significant t-test results
dt_t_test_data <-
t_test_results %>%
  filter(response == "recovery") %>%
  mutate(star = case_when(
    p_value < 0.001 ~ "***",
    p_value < 0.01 ~ "**",
    p_value < 0.05 ~ "*",
    TRUE ~ "ns"
    )) %>%
  select(species, star) %>%
  left_join(select(traits, species, generation, recovery_mean, recovery_sd), by = "species") %>%
  rowwise %>%
  mutate(recovery_top = sum(recovery_mean, recovery_sd)) %>%
  group_by(species, star) %>%
  summarize(recovery_top = max(recovery_top), .groups = "drop")

# Set position dodge for bars
pos <- position_dodge(
  width = 0.5, # Bars right next to each other
  preserve = "single" # Keep bar width constant
  )

b <- ggplot(traits, aes(y = species)) +
  geom_col(
    aes(x = recovery_mean, fill = generation),
    position = pos,
    width = 0.5) +
  geom_errorbarh(
    aes(x = recovery_mean, xmax = recovery_mean + recovery_sd, xmin = recovery_mean, fill = generation),
    position = pos,
    height = 0,
    color = "grey20") +
  geom_text(
    data = dt_t_test_data, 
    aes(label = star, y = species, x = recovery_top),
    angle = -90,
    nudge_x = 0.01,
    color = "grey20",
    size = 2.5,
    vjust = -0.5) +
  scale_x_continuous(
    labels = percent, 
    limits = c(0,1.2), 
    expand = expansion(mult = c(0,0.030)), 
    breaks=c(0,.25,.5,.75,1.0)) +
  scale_fill_manual(values = generation_colors, labels = pretty_text) +
  # theme_bw(base_size = 10) +
  labs(x = "Recovery", fill = "Generation") +
  theme(
    panel.grid.minor = element_blank(), 
    panel.grid.major.y = element_blank(),
    axis.title.y = element_blank(),
    axis.text.y = element_blank(),
    axis.ticks.y = element_blank(),
    legend.position = "bottom"
  )

# Part C (right): light-levels

# Make helper table of significant t-test results: ETR
etr_t_test_data <-
t_test_results %>%
  filter(response == "etr") %>%
  mutate(star = case_when(
    p_value < 0.001 ~ "***",
    p_value < 0.01 ~ "**",
    p_value < 0.05 ~ "*",
    TRUE ~ "ns"
    )) %>%
  select(species, star) %>%
  left_join(select(traits, species, generation, etr_mean, etr_sd), by = "species") %>%
  rowwise %>%
  # `etr_top` is the highest possible value of ETR in the plot (mean + sd),
  # used for placing the asterisks
  mutate(etr_top = sum(etr_mean, etr_sd)) %>%
  group_by(species, star) %>%
  summarize(etr_top = max(etr_top), .groups = "drop")

# Make helper table of significant t-test results: PAR
par_t_test_data <-
t_test_results %>%
  filter(response == "par") %>%
  mutate(star = case_when(
    p_value < 0.001 ~ "***",
    p_value < 0.01 ~ "**",
    p_value < 0.05 ~ "*",
    TRUE ~ "ns"
    )) %>%
  select(species, star) %>%
  left_join(select(traits, species, generation, par_mean, par_sd), by = "species") %>%
  rowwise %>%
  mutate(par_top = sum(par_mean, par_sd)) %>%
  group_by(species, star) %>%
  summarize(par_top = max(par_top), .groups = "drop")

c <- ggplot(traits, aes(y = species)) +
  geom_col(
    aes(x = etr_mean, fill = generation),
    position = pos,
    width = 0.5) +
  geom_errorbarh(
    # need `fill = generation` for position to work correctly,
    # even though it produces a warning 'Ignoring unknown aesthetics: fill '
    aes(xmax = etr_mean + etr_sd, xmin = etr_mean, fill = generation),
    position = pos,
    height = 0,
    color = "grey20") +
  geom_text(
    data = etr_t_test_data, 
    aes(label = star, y = species, x = etr_top),
    angle = -90,
    nudge_x = 0.01,
    color = "grey20",
    size = 2.5,
    vjust = -0.5) +
  scale_fill_manual(values = generation_colors, labels = pretty_text) +
  scale_y_discrete(
    position = "right",
    # Format species names by converting underscore to space and italicizing
    labels = function(x) str_replace_all(x, "_", " ") %>% paste0("*", ., "*")
    ) +
  scale_x_continuous(expand = expansion(mult = c(0,0.025))) +
  labs(x = "ETR<sub>max</sub>", fill = "Generation") +
  theme(
    panel.grid.minor = element_blank(), 
    panel.grid.major.y = element_blank(),
    axis.title.y = element_blank(),
    axis.text.y = element_blank(),
    axis.ticks.y = element_blank(),
    axis.title.x = element_markdown()
  ) +
  guides(fill = "none")


d <- ggplot(traits, aes(y = species)) +
  geom_col(
    aes(x = par_mean, fill = generation),
    position = pos,
    width = 0.5) +
  geom_errorbarh(
    # need `fill = generation` for position to work correctly,
    # even though it produces a warning 'Ignoring unknown aesthetics: fill '
    aes(xmax = par_mean + par_sd, xmin = par_mean, fill = generation),
    position = pos,
    height = 0,
    color = "grey20") +
  geom_text(
    data = par_t_test_data, 
    aes(label = star, y = species, x = par_top),
    angle = -90,
    nudge_x = 0.01,
    color = "grey20",
    size = 2.5,
    vjust = -0.5) +
  scale_fill_manual(values = generation_colors, labels = pretty_text) +
  scale_y_discrete(
    position = "right",
    labels = str_abbrev_sp_pretty
    ) +
  scale_x_continuous(expand = expansion(mult = c(0,0.025))) +
  labs(x = "PPFD<sub>95%</sub>", fill = "Generation") +
  theme(
    panel.grid.minor = element_blank(), 
    panel.grid.major.y = element_blank(),
    axis.title.y = element_blank(),
    axis.text.y.right = element_markdown(), # NOT axis.text.y
    axis.title.x = element_markdown()
  ) +
  guides(fill = "none")

# Make combined plot
combined_plot <- a + b + c + d + plot_layout(nrow = 1, guides = "collect") &
  theme(
    legend.position = "bottom",
    # Adjust invididual margins to remove whitespace between subplots
    plot.margin = margin(t = 0.1, r = 0, b = 0.1, l = 0.04, unit = "in"),
    # Wrap legend parts into multiple lines
    legend.box = "vertical", 
    legend.margin = margin())
```

```{r combined-plot-save, eval = params$output_type == "ms", purl = FALSE}
# Save plot to file if knitting MS for submission
combined_plot_name <- figure("combined") %>% str_remove_all("[:punct:]| ") %>% paste0(".pdf")
ggsave(plot = combined_plot, file = here::here("results/ms", combined_plot_name), width = 6.85, height = 5)
```

```{r combined-plot-print, fig.width = 6.85, fig.height = 5, fig.cap = figure("combined"), eval = params$output_type == "preprint", purl = FALSE}
# Print plot if knitting MS to preprint
combined_plot
```

```{r corr-plot-make, fig.width = 6.85, fig.height = 5, fig.cap = "", purl = FALSE}
# Make plot

# Combine recovery, VPD, and growth habit into a single dataframe (wide format)
recovery_mean_data <- select(combined_species_means, species, generation, recovery_mean) %>%
  pivot_wider(values_from = recovery_mean, names_from = generation) %>%
  rename_with(~glue("recovery_{.}_mean"), .cols = -species)

recovery_sd_data <- select(combined_species_means, species, generation, recovery_sd) %>%
  pivot_wider(values_from = recovery_sd, names_from = generation) %>%
  rename_with(~glue("recovery_{.}_sd"), .cols = -species)
  
plot_data <-
list(filmy_habit, recovery_mean_data, recovery_sd_data, mean_vpd_sporo, mean_vpd_gameto, range_comparison) %>%
  reduce(left_join, by = "species")

# Verify that model fit is significant
vpd_sporophyte_pval <- env_range_dt_model_summary %>% filter(model == "sporo_vpd_model") %>% pull(p_value) %>%
  round_t(3)

assertthat::assert_that(vpd_sporophyte_pval < 0.05)

vpd_sporophyte_adj_r_squared <- env_range_dt_model_summary %>% filter(model == "sporo_vpd_model") %>% pull(adj_r_squared) %>%
  round_t(3)

# Extract fitted points from model
vpd_sporophyte_fitted <-
  # PGLS model has fitted points by species name, so join with VPD data
  env_range_dt_model$sporo_vpd_model$fitted %>%
  as.data.frame() %>%
  rownames_to_column("species") %>%
  as_tibble() %>%
  rename(recovery_sporophyte_mean = V1) %>%
  left_join(select(
    plot_data, species, vpd_sporophyte_mean
  ), by = "species")

a <- ggplot(plot_data, aes(x = vpd_sporophyte_mean, y = recovery_sporophyte_mean, color = habit)) +
  geom_line(data = vpd_sporophyte_fitted, color = "dark grey") +
  geom_linerange(aes(
    ymin = recovery_sporophyte_mean - recovery_sporophyte_sd,
    ymax = recovery_sporophyte_mean + recovery_sporophyte_sd),
    alpha = 0.3) +
  geom_linerange(aes(
    xmin = vpd_sporophyte_mean - vpd_sporophyte_sd,
    xmax = vpd_sporophyte_mean + vpd_sporophyte_sd),
    alpha = 0.3) +
  geom_point() +
  annotate(
    geom = "richtext", 
    x = Inf, y = -Inf, 
    label = glue("*P* = {vpd_sporophyte_pval}<br>adj. *R*<sup>2</sup> = {vpd_sporophyte_adj_r_squared}"),
    hjust = 1, vjust = 0, size = 3, label.colour = "transparent") +
  scale_y_continuous(
    labels = percent, 
    limits = c(0,1.25)) +
  scale_color_manual(values = habit_colors, labels = pretty_text) +
  labs(
    y = "Recovery",
    x = "VPD (kPa)",
    color = "Growth habit")

b <- ggplot(plot_data, aes(x = vpd_gametophyte_mean, y = recovery_gametophyte_mean, color = habit)) +
  geom_linerange(aes(
    ymin = recovery_gametophyte_mean - recovery_gametophyte_sd,
    ymax = recovery_gametophyte_mean + recovery_gametophyte_sd),
    alpha = 0.3) +
  geom_linerange(aes(
    xmin = vpd_gametophyte_mean - vpd_gametophyte_sd,
    xmax = vpd_gametophyte_mean + vpd_gametophyte_sd),
    alpha = 0.3) +
  geom_point() +
  scale_y_continuous(
    labels = percent, 
    limits = c(0,1.25)) +
  scale_color_manual(values = habit_colors, labels = pretty_text) +
  labs(
    y = "Recovery",
    x = "VPD (kPa)",
    color = "Growth habit") +
  guides(color = "none")

c <- ggplot(plot_data, aes(x = gameto_beyond_sporo, y = recovery_gametophyte_mean, color = habit)) +
  geom_linerange(aes(
    ymin = recovery_gametophyte_mean - recovery_gametophyte_sd,
    ymax = recovery_gametophyte_mean + recovery_gametophyte_sd),
    alpha = 0.3) +
  geom_point() +
  scale_color_manual(values = habit_colors, labels = pretty_text) +
  scale_y_continuous(labels = percent) +
  labs(
    y = "Recovery",
    x = "Elevation (m)",
    color = "Growth habit") +
  guides(color = "none")

corr_plot <- a + b + c + 
  guide_area() + 
  plot_layout(guides = "collect") +
  plot_annotation(tag_levels = "a") &
  theme(plot.tag = element_text(face = 'bold'))
```

```{r corr-plot-save, eval = params$output_type == "ms", purl = FALSE}
# Save plot to file if knitting MS for submission
corr_plot_name <- figure("corr") %>% str_remove_all("[:punct:]| ") %>% paste0(".pdf")
ggsave(plot = corr_plot, file = here::here("results/ms", corr_plot_name), width = 6.85, height = 5)
```

```{r corr-plot-print, fig.width = 6.85, fig.height = 5, fig.cap = figure("corr"), eval = params$output_type == "preprint", purl = FALSE}
# Print plot if knitting MS to preprint
corr_plot
```
